.eslintrc

```diff
@@ -0,0 +1,20 @@
+---
+extends: react-app
```

Comment:
In order to use the same rules in the server package (Which doesn't use CRA) I had to add some dependencies

```json
  "devDependencies": {
    "babel-eslint": "^7.2.3",
    "eslint": "^3.19.0",
    "eslint-config-react-app": "^1.0.5",
    "eslint-plugin-flowtype": "^2.33.0",
    "eslint-plugin-import": "^2.2.0",
    "eslint-plugin-jsx-a11y": "^5.0.3",
    "eslint-plugin-react": "^7.0.1",
    "nodemon": "^1.11.0"
  }
```